components/dialog/dialog-fullscreen.js

```diff
@@ -25,68 +25,134 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 			 * Whether to render a loading-spinner and wait for state changes via AsyncContainerMixin
 			 */
 			async: { type: Boolean },
-			_hasFooterContent: { type: Boolean, attribute: false }
+			_hasFooterContent: { type: Boolean, attribute: false },
+			_iconType: { type: String, attribute: false }
```

Comment:
Nit: I'd just call this `_icon`, to be consistent with how we name this property elsewhere, and also since it's not really a type of icon.